docs/src/_reference/plugin-definition-syntax.md

```diff
@@ -460,7 +460,7 @@ settings:
 
 ### `settings[*].kind`
 
-Optional. Use for a first-class input control. Default is `string`, others are `integer`, `boolean`, `date_iso8601`, `password`, `options`, `file`, `array`, `object`, and `hidden` (deprecated).
+Optional. Use for a first-class input control. Default is `string`, others are `integer`, `boolean`, `date_iso8601`, `password` (deprecated), `options`, `file`, `array`, `object`, and `hidden` (deprecated).
```

Comment:
```suggestion
Optional. Use for a first-class input control. Default is `string`, others are `integer`, `boolean`, `date_iso8601`,, `options`, `file`, `array`, `object`.  

`password` and `hidden` are both deprecated.
```

Reply:
Partially addressed this in 6e68c3e19c2e93b8f93ffb1708d2f3a005fa6f09 - deprecated kinds are already listed below in a warning block.